deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -115,13 +119,44 @@ extern "C" __declspec(noreturn) void __fastfail(unsigned int);
 #define TAEF_FAST_FAIL System::Environment::FailFast(nullptr)
 #endif // #if !defined(__cplusplus_cli)
 
+#if defined(__clang__)
+#include <array>
+#endif
+
 /// <summary>
 /// The WEX namespace contains classes used for marking up C++ Tests.
 /// </summary>
 namespace WEX
 {
     namespace Private
     {
+        #if defined(__clang__)
+        // tempalated struct using variadic template args to represent a sequence of characters
+        template <unsigned... CharacterIndices> 
+        struct index_sequence{};
```

Comment:
Brian's copying the standard names, so I'd be fine with just a comment. std::index_sequence should be in 'utility' - should you use that? TBH, I'm torn as to whether you should #include utility or not. It's always been painful to remove #includes, since downstream consumers get grumpy if they were getting them for free; so absolutely minimizing the #includes is typically a priority. Anyone else have comments?